### ISSUE TITLE
Separators breaking functionality 

### ISSUE THREAD

**Issue by gazny** [budgie-desktop/budgie-desktop#1030]
**Operating System**:  Solus 2017.04.18.0 shannon
**Budgie version**: budgie-desktop v10.3.1-367-g8782f5b

**The issue encountered**
Budgie applets settings incorectly displays and blocks the abilitiy to move applets down(raven sidebar control)
 ...

**Steps to reproduce the issue**
Moving separators  at the bottom of the top panel up and down in settings
![screenshot from 2017-07-13 12 27 35](https://user-images.githubusercontent.com/19635950/28184059-b282af18-67c6-11e7-8f69-aecf494aab16.png)

 ...

**Comment by ikeydoherty** [comment]
I didn't understand any of that. But those up/down buttons are sensitive and those separators can be removed (the buttons are lit)

**Comment by gazny** [comment]
It works fine without separators, but when I introduce them the positioning of the raven menu and others position's are locked and can not be moved down, and can only be moved up with the arrows.

**Comment by tristan957** [comment]
@ikeydoherty I think @gazny is talking about how the down arrow is inactive even though the user indicator could move under the Raven Sidebar Control. I may be wrong though

**Comment by gazny** [comment]
@tristan957 Yes, that is exactly it. Only happens when separators are present without spacers
